README.md

```diff
@@ -1,210 +1,305 @@
+beginners. Once used to mod-taps, you may want to reduce it so keyboard
+shortcuts can be done more quickly.
 
-Other desktop implementations (kmonad, keyd…) would be nice to see as well.
+In the <kbd>NumRow</kbd> layer, you can edit the <kbd>dk1</kbd> to 
+<kbd>dk5</kbd> shortcuts to put whatever seams useful to you, a lot of available
+keys are defined in [Kanata source code][Kanata keys].
 
-Programmable keyboards should be trivial to configure with QMK, ZMK,
-Kaleidoscope, etc.
+In the <kbd>Navigation</kbd> layer, you can put a command on top of the
+<kbd>P</kbd> key (in Qwerty), e.g. for an application launcher.
 
+Note that Kanata can also use the laptop’s trackpoint buttons (e.g. ThinkPad)
+as two additional thumb keys. :-)
 
-Related Projects
+
+Installation
 --------------------------------------------------------------------------------
 
+Adjusting to compact keyboard layouts isn’t easy, but Arsenik is designed for
+a step-by-step approach:
+
+- load `kanata.kbd` with Kanata ([installation instructions](kanata))
+- enable each feature by un-commenting the related line (a commented line starts
+with `;;`), you must enable one and only one line per feature
+- live-reload the configuration with <kbd>Space</kbd>+<kbd>Backspace</kbd>
+(requires the feature layer-taps enabled)
+
+If you have a programmable keyboard you might want to take a look at the
+[QMK](qmk) version of Arsenik (work in progress).
+
+Other desktop implementations (kmonad, keyd, Karabiner…) would be nice to see as
+well.
+
+
+Why “Arsenik”?
+--------------------------------------------------------------------------------
+
+33 keys layout: the 33rd element of the periodic table.
+
+Unlike Miryoku which requires 6 thumb keys, Arsenik has been designed to work
+with standard ANSI/ISO/laptop keyboards, leveraging the spacebar and the two
+Alt/Cmd keys.
+
 ### Inspiration
 
-- [Miryoku][1] for the main idea of using modifiers on the homerow and layer
+- [Miryoku] for the main idea of using modifiers on the home row and layer
 shifters under the thumbs;
-- [Lafayette][2] and [Ergo-L][3] for the <kbd>Symbol</kbd> layer, which has been
-blatantly taken *as is*;
-- [Extend][4], [Neo][5], [Shaka34][6] for the <kbd>Navigation</kbd> layer.
+- [Lafayette] and [Ergo-L] for the <kbd>Symbol</kbd> layer, which has been
+shamelessly taken *as is*;
+- [Extend], [Neo], [Shaka34] for the <kbd>Navigation</kbd> layer.
 
 ### Alternative Symbol Layers
 
-- [Neo][5]
-- [Seniply][7]
-- [Pascal Getreuer’s][8]
+- [Neo]
+- [Seniply]
+- [Pascal Getreuer’s]
 
 ### Non-Goals
 
-- being the most efficient 3×5 layout — [Miryoku][1] is probably the most
-advanced approach for that, at least on custom 36-key keyboards;
+- being the most efficient 3×5 layout — [Miryoku] is probably the most
+advanced approach for that, at least on custom 36-key keyboards; and each person
+its own configuration for their needs will be more efficient, even if Arsenik
+can serve at a starting point
```

Comment:
I’d suggest to replace the sub-sentence after the semicolon by a bullet point, e.g.:

```md
- suiting every user out-of-the-box — Arsenik is proposed as a reasonable default configuration,
but users are encouraged to customize it to suit their personal needs and preferences;
```

Reply:
Agreed, it’s a lot clearer to read. I’ve changed this according to what you suggest.